packages/medusa/src/services/__tests__/discount.js

```diff
@@ -52,6 +51,24 @@ describe("DiscountService", () => {
       }
     })
 
+    it("fails to create a discount without regions", async () => {
+      expect.assertions(3)
```

Comment:
todo: same as the other tests, `const err = await discountService.create().catch(e => e)` and you can remove the try catch and the expect.assertions since it will always reach the expects